bot/cogs/stats.py

```diff
@@ -37,7 +37,14 @@ def __init__(self, bot):
     @commands.command(brief='See your stats')
     async def stats(self, ctx):
         """ Send an embed containing stats data parsed from the player object returned from the API. """
-        user = ctx.author
+        
+        try:
+            user = ctx.message.mentions[0]
+        except IndexError:
+            user = ctx.author
+        else:
```

Comment:
The `else` statement here seems unnecessary, because if there aren't any exceptions, then `user` will already be set to `ctx.message.mentions` from the `try` block.